### PR TITLE
Add support for PureScript 0.9.1

### PR DESCRIPTION
**Pull request by paulyoung**
Please review 🙏

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-aff": "^0.16.0",
-    "purescript-free": "^0.9.1"
+    "purescript-aff": "~1.0.0",
+    "purescript-free": "~1.0.0"
```
Comment (garyb) [inline bower.json]: Can you use `^` ranges here please?

File: package.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
     "build": "pulp build"
   },
   "devDependencies": {
-    "pulp": "^8.1.0",
-    "purescript": "^0.7.6",
+    "pulp": "^9.1.0",
```
Comment (garyb) [inline package.json]: `pulp` has only reached 9.0.1 so far :smile: (updating this should fix Traivs)

**Comment by paulyoung** [comment]
@garyb thanks for catching those. I amended the commit and force pushed.

**Comment by garyb** [comment]
Thanks!

**Pull request merged**